assets/schema_input.json

```diff
@@ -38,6 +38,18 @@
                 "errorMessage": "Sample type must be provided and be one of 'riboseq', 'rnaseq' or 'tiseq'",
                 "enum": ["riboseq", "rnaseq", "tiseq"],
                 "meta": ["sample_type"]
+            },
+            "trim_length": {
+                "anyOf": [
+                    { "type": "integer", "minimum": 15 },
+                    { "type": "string", "pattern": "^[0-9]+$" }
+                ],
+                "errorMessage": "Trim length must be a positive integer of at least 15",
```

Comment:
I'm not sure I understand why we need a min value here. I guess nobody should be bothered to use this if the lenght difference is less than 15.